node/src/service.rs

```diff
@@ -40,73 +38,42 @@ use sp_keystore::KeystorePtr;
 use sp_runtime::{app_crypto::AppCrypto, traits::BlakeTwo256};
 use substrate_prometheus_endpoint::Registry;
 
-/// Devnet Native executor type.
-pub struct DevnetRuntimeExecutor;
-
-impl NativeExecutionDispatch for DevnetRuntimeExecutor {
-	type ExtendHostFunctions = (
-		cumulus_client_service::storage_proof_size::HostFunctions,
-		frame_benchmarking::benchmarking::HostFunctions,
-	);
-
-	fn dispatch(method: &str, data: &[u8]) -> Option<Vec<u8>> {
-		pop_runtime_devnet::api::dispatch(method, data)
-	}
-
-	fn native_version() -> sc_executor::NativeVersion {
-		pop_runtime_devnet::native_version()
-	}
-}
-
-/// Testnet Native executor type.
-pub struct TestnetRuntimeExecutor;
+type HostFunctions = (
+	cumulus_client_service::ParachainHostFunctions,
+	frame_benchmarking::benchmarking::HostFunctions,
+);
```

Comment:
Thanks for adding this back! Can we please feature-gate for if benchmarks / if not
```suggestion
#[cfg(not(feature = "runtime-benchmarks"))]
type HostFunctions = cumulus_client_service::ParachainHostFunctions;

#[cfg(feature = "runtime-benchmarks")]
type HostFunctions = (
	cumulus_client_service::ParachainHostFunctions,
	frame_benchmarking::benchmarking::HostFunctions,
);
```

Reply:
@peterwht Thank you, I did saw that in the other referenced PR but as I read: "After you have once introduced a host function that was used by any runtime, you can not remove it." (https://substrate.stackexchange.com/a/3162/703) so I wonder if `frame_benchmarking::benchmarking::HostFunctions` must be added by default without the feature gate.  Hope you can give me more insight!